drop_all_tables.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+func dropAllTablesCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "drop-all-tables",
+		Short: "Drop all tables",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			dbName, err := dbNameFromFlagEnv()
+			if err != nil {
+				return err
+			}
+			log.Printf("db: %s\n", dbName)
+
+			ctx := context.Background()
+
+			admin, cli := createClients(ctx, dbName)
+			tables, err := query(ctx, cli, "SELECT TABLE_NAME FROM information_schema.tables WHERE TABLE_CATALOG = '' AND TABLE_SCHEMA = ''")
+			if err != nil {
+				return err
+			}
+			for _, t := range tables.rows {
+				indexes, err := query(ctx, cli, "SELECT INDEX_NAME FROM information_schema.indexes WHERE INDEX_NAME != 'PRIMARY_KEY' AND TABLE_NAME='"+t[0]+"'")
+				if err != nil {
+					return err
+				}
+				for _, i := range indexes.rows {
+					log.Printf("Dropping index: %s\n", i[0])
+					if err := updateDatabaseDdl(ctx, os.Stdout, admin, dbName, "DROP INDEX "+i[0]); err != nil {
+						return err
+					}
+				}
+				log.Printf("Dropping table: %s\n", t[0])
+				if err := updateDatabaseDdl(context.Background(), os.Stdout, admin, dbName, "DROP TABLE "+t[0]); err != nil {
+					return err
+				}
+			}
```

Comment:
This approach would have a problem with dependency.

### Foreign Key constraint.
If a table has one or more foreign key constraints referencing another table, you can't drop the table before you drop the referenced table or the foreign key.

```sql
CREATE TABLE Orders (
      OrderID INT64 NOT NULL,
      CustomerID INT64 NOT NULL,
      Quantity INT64 NOT NULL,
      ProductID INT64 NOT NULL,
      CONSTRAINT FK_CustomerOrder FOREIGN KEY (CustomerID) REFERENCES Customers (CustomerID)
    ) PRIMARY KEY (OrderID);
```

### Interleaved tables.
Tables can have one or more child (interleaved) tables. In this case, you can't drop the parent table before you drop the children. Worse yet, children tables can have their children...

We need to resolve those dependencies.  We might need to build a tree of tables and walk through from the most-children to the root.

```sql
CREATE TABLE Root ( ID INT64 NOT NULL ) PRIMARY KEY(ID);
CREATE TABLE Child ( ID INT64 NOT NULL ) PRIMARY KEY(ID), INTERLEAVE IN PARENT Root;
CREATE TABLE GrandChild (ID INT64 NOT NULL) PRIMARY KEY(ID), INTERLEAVE IN PARENT Child; 
```

By the way, executing DDL is so slow. I'm going to push a batch execution version of updateDatabaseDdl. It will be way much faster than calling updateDatabaseDdl many times... And then, I think my change will conflict with this branch and difficult to merge it. Don't we revert the split of the main.go and keep the change minimum? Adding drop_all_tables.go looks nice. I will split `main.go` after this PR.